### DISCUSSION TITLE
Migration from UWP

### DISCUSSION THREAD

**Discussion by mracavano** [PrismLibrary/Prism#2895] [General]
Hi,

Are there options to upgrade a UWP App that uses Prism and Unity to versions of Prism and any IoC that are not deprecated?  

Thank you

**Comment by brianlagunas** [comment]
How are you using Prism with UWP? Prism doesn't support UWP unless you're using Uno.

**Reply by mracavano** [reply]
Currently using Prism.Windows v6.3 that seems to have supported UWP.

**Reply by mracavano** [reply]
I looked at Uno but without Syncfusion and Telerik support for that platform, things would be difficult.

**Reply by brianlagunas** [reply]
Prism.Windows has been dead since March 2017. There is no direct replacement for it.

**Reply by mracavano** [reply]
Is moving to WinUI or MAUI an option?

**Reply by brianlagunas** [reply]
Moving, no. Rewrite, yes.

**Reply by mracavano** [reply]
Considering Xamrin Forms, Maui, WinUI:
Are all supported by stable releases of Prism?
Do you think one has a smaller re-write burden?
Do you think one has better long term support odds?
Thank you Brian

**Reply by brianlagunas** [reply]
Microsoft killed XF, and basically killed WinUI too. So, the only one left standing is Maui.
